### ISSUE TITLE
Scratch on amputated limb

### ISSUE THREAD

**Issue by RubyRedsword** [ZioPao/The-Only-Cure#67]
hey I'm pretty sure I'm not running an older version but it seems I've gotten scratched again, feel free to tell me I'm wrong though!
**Logs**
[console.txt](https://github.com/ZioPao/The-Only-Cure/files/15285640/console.txt)

**Additional context**
![20240512045456_1](https://github.com/ZioPao/The-Only-Cure/assets/168946486/b2d10a8a-5cd1-4c57-8b7e-deb0e365bde4)

**Comment by ZioPao** [comment]
You're right, you're running the right version. Strange, I'll keep looking into it

**Comment by RubyRedsword** [comment]
Any new yet?

**Comment by ZioPao** [comment]
No, sorry, I didn't have time to work on it

**Comment by SebbyLikesCats** [comment]
> No, sorry, I didn't have time to work on it
> 
> By any chance, did you die before the problem occurred? If so, I think I may have found the issue.

I had the same issue and I had  2 characters before the bug happened

**Comment by RubyRedsword** [comment]
I did not, it was  brand new save

**Comment by RubyRedsword** [comment]
hey i know life has probably been busy but any news?

**Comment by criticap** [comment]
I'm not sure how hard is it for the solution, but someone actually come up with a solution of giving the prosthetic arm item a 100% protection and this problem is solved.
However, having 100% protection will protect both arms instead of that one specific arm.
But hey, it is a solution nonetheless for an issue back in May...
